.github/workflows/pull_request.yaml

```diff
@@ -18,9 +18,15 @@ on:
         type: boolean
         required: false
         default: false
+      runHelmLint:
```

Comment:
```suggestion
      enableHelmLint:
```

Reply:
@shimoncohen why `runHelmLint` is not good??